src/hooks/useJwtDecoder.tsx

```diff
@@ -4,9 +4,13 @@ import jwtDecode, { JwtPayload } from "jwt-decode";
 const useJwtDecoder = () => {
   const { authToken } = useAuth();
 
-  const { sub, exp } = jwtDecode<JwtPayload>(authToken!);
+  if (authToken) {
+    const { sub, exp } = jwtDecode<JwtPayload>(authToken);
 
-  return { sub, exp };
+    return { sub, exp };
+  }
+
+  return { sub: "0", exp: "0" };
 };
```

Comment:
I dont like this function, I dont see reason to use it

Reply:
This one returns the id and expiration date stored in the JWT token, I believe it will be useful in the future to implement auto logout after token expiration.